packages/block-library/src/navigation/edit/placeholder/index.js

```diff
@@ -34,13 +34,14 @@ const ExistingMenusDropdown = ( {
 	onCreateFromMenu,
 } ) => {
 	const toggleProps = {
-		variant: 'primary',
+		variant: 'tertiary',
+		iconPosition: 'right',
 		className: 'wp-block-navigation-placeholder__actions__dropdown',
 	};
 	return (
 		<DropdownMenu
-			text={ __( 'Select existing menu' ) }
-			icon={ chevronDown }
+			text={ __( 'Existing menu' ) }
```

Comment:
Could we keep "Select existing menu"? My fear is that "Existing menu" is a bit too cryptic. The other buttons are clearer in their intent with "Add...", "Create..." etc.

Reply:
I was looking at it yesterday, and wondered if it could be shortened to just 'Select menu'.